### ISSUE TITLE
Sever down

### ISSUE THREAD

**Issue by yashha** [Rostlab/JS16_ProjectA#94]
@kordianbruck  please start server :)

**Comment by bandle** [comment]
We have extracted the data out of wikiDate folder and pushed it into our own git:
https://github.com/Rostlab/JS16_ProjectC_Group10/tree/develop/data

For the moment you can use those to get some got data into your page. 
The file should be similar to the DBs out when calling getAll

**Comment by ghost** [comment]
@yashha @mbandle 
Sorry, but the data of wikiData/ are not similar to the db. Many properties are not matched against the models and transformed. =/

**Comment by bandle** [comment]
I added some extra checks and it works for our map. At least its better than no data at all …

**Comment by sacdallago** [comment]
@yashha get used to using the API locally for testing! @Adiolis please provide them with the config that connects to mlab

**Comment by ghost** [comment]
@mbandle @yashha 
Please give me hints how i can reach you guys to provide the config.
The mLab db is most of time provided with the newest stuff :P 
Unfortunately you will need to run the server locally until kordians public server is on again...

**Comment by julienschmidt** [comment]
@Adiolis can you please also send me the config via email:  my.name at tum.de

**Comment by sacdallago** [comment]
Keep me posted on the progress here @julienschmidt @Adiolis ... Right now @julienschmidt is my versuchskaninchen

**Comment by ghost** [comment]
@julienschmidt you should have an email.
I will see that the db is in the correct state after i created the new scripts for filling the db. #95

**Comment by julienschmidt** [comment]
Assuming that ~~#98 is merged and~~ mongodb installed and running:

``` sh
git clone https://github.com/Rostlab/JS16_ProjectA.git
cd JS16_ProjectA
# copy config.json to /cfg
npm install
npm install apidoc -g
apidoc -i app/ -o apidoc/
npm start
```

werks

~~ProTipp™: set `"fillerPolicy": 1` in the config.json(#95)~~

**Comment by julienschmidt** [comment]
All those `sudo` commands in the README shouldn't be necessary btw. If you need them, there is something wrong with your node setup.

**Comment by kordianbruck** [comment]
I can give you access to the mLab which is used by my server. Just write me a mail (you can find this in my profile)

Server is up. Someone added a new dependency and didn't tell me about it. Also: please don't spam random issues with problems you encounter, it makes it really hard to follow what is required.
